adr/0002-workflow-integration-points.md

```diff
@@ -0,0 +1,35 @@
+# 2. Provide workflow integration points
+
+Date: 2018-04-27
+
+## Status
+
+Proposed
+
+## Context
+
+A system to handle the creation, processing and orchestration of customized workflows is a required component of Libero's
+currently proposed design.
+
+Users will each have their individual needs and requirements and therefore may decide to use their 
+workflow solution of choice. 
+
+It is therefore important to define clear integration points between a workflow system and the rest of the Libero
+architecture.
+
+## Decision
+
+The workflow solution should provide:
+
+- HTTP endpoint(s) to allow the triggering of workflows
+- A queue (AMQP 0.9.1) to allow the triggering of workflows
```

Comment:
I think a queue is implementation-specific, for compatibility any interaction with the workflow system with the could be through a standard HTTP API